notebooks/MetPy SkewT.ipynb

```diff
@@ -0,0 +1,442 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = plt.figure(figsize=(9, 11))\n",
+    "skew = SkewT(fig, rotation=45)\n",
+    "\n",
+    "skew.plot(p, T, 'r')\n",
+    "skew.plot(p, Td, 'g')\n",
+    "skew.plot_barbs(p[::3], u[::3], v[::3], y_clip_radius=0.03)\n",
+    "\n",
+    "skew.ax.set_xlim(-30, 40)\n",
+    "skew.ax.set_ylim(1020, 100)\n",
+    "\n",
+    "skew.plot_dry_adiabats(alpha=0.25, color='orangered')\n",
+    "skew.plot_moist_adiabats(alpha=0.25, color='tab:green')\n",
+    "skew.plot_mixing_lines(linestyle='dotted', color='tab:blue')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, let's add a title to the plot"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = plt.figure(figsize=(9, 11))\n",
+    "skew = SkewT(fig, rotation=45)\n",
+    "\n",
+    "skew.plot(p, T, 'r')\n",
+    "skew.plot(p, Td, 'g')\n",
+    "skew.plot_barbs(p[::3], u[::3], v[::3], y_clip_radius=0.03)\n",
+    "\n",
+    "skew.ax.set_xlim(-30, 40)\n",
+    "skew.ax.set_ylim(1020, 100)\n",
+    "\n",
+    "skew.plot_dry_adiabats(alpha=0.25, color='orangered')\n",
+    "skew.plot_moist_adiabats(alpha=0.25, color='tab:green')\n",
+    "skew.plot_mixing_lines(linestyle='dotted', color='tab:blue')\n",
+    "\n",
+    "plt.title('{} Sounding'.format(station), loc='left')\n",
+    "plt.title('Valid Time: {}'.format(dt), loc='right')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Add a hodograph\n",
+    "\n",
+    "In this section, we'll add a hodograph inset to the plot.\n",
+    "Everything above the two commented lines is the same as the previous skew-T plot. The next two lines add a hodograph inset using matplotlib's inset_axes functionality and MetPy's Hodograph class."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = plt.figure(figsize=(9, 11))\n",
+    "skew = SkewT(fig, rotation=45)\n",
+    "skew.plot(p, T, 'r')\n",
+    "skew.plot(p, Td, 'g')\n",
+    "skew.plot_barbs(p[::3], u[::3], v[::3], y_clip_radius=0.03)\n",
+    "skew.ax.set_xlim(-30, 40)\n",
+    "skew.ax.set_ylim(1020, 100)\n",
+    "skew.plot_dry_adiabats(alpha=0.25, color='orangered')\n",
+    "skew.plot_moist_adiabats(alpha=0.25, color='tab:green')\n",
+    "skew.plot_mixing_lines(linestyle='dotted', color='tab:blue')\n",
+    "plt.title('{} Sounding'.format(station), loc='left')\n",
+    "plt.title('Valid Time: {}'.format(dt), loc='right')\n",
+    "#########################################################\n",
+    "#########################################################\n",
```

Comment:
Remove all code after the '###' lines, for this cell block.
Remove the two lines of '###' as well.